Formula/fortio.rb

```diff
@@ -0,0 +1,39 @@
+class Fortio < Formula
+  desc "HTTP and gRPC load testing and visualization tool and server"
+  homepage "https://github.com/istio/fortio"
+  url "https://github.com/istio/fortio/archive/v0.9.0.tar.gz"
+  sha256 "76871124e7abd13d7fa0ccc5ad6912bc1c3d77d53694da2b26ad44af3638edab"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = "amd64"
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
why?

Reply:
Removed the CGO_ENABLED flag without any issues.
@ldemailly FYI

Reply:
CGO_ENABLED=0 is to get truly static binaries that do not depend on go installation; I would think it is necessary (or at least does not hurt, the fortio binary is small)

Reply:
> CGO_ENABLED=0 is to get truly static binaries that do not depend on go installation

This isn't the case for most software, why is it for Fortio?



> or at least does not hurt

We don't want to add things unless they are 100% necessary.

Reply:
thanks. can you point me to another go program tap as an example ?
(fortio ships as a static binary for docker and linux to be "copyable" I thought it's a useful feature for mac os as well, I am not sure how it even works dynamically on a host that doesn't have go runtime installed)

Reply:
@ldemailly https://github.com/Homebrew/homebrew-core/blob/master/Formula/fork-cleaner.rb. You shouldn't have to remove `CGO` support for it to not depend on `go` at runtime.